clients/python/tests/conftest.py

```diff
@@ -29,13 +29,14 @@ def pytest_collection_modifyitems(config, items):
             continue
 
 
-REGISTRY_HOST = "http://localhost"
-REGISTRY_PORT = 8080
-REGISTRY_URL = f"{REGISTRY_HOST}:{REGISTRY_PORT}"
-COMPOSE_FILE = "docker-compose.yaml"
-MAX_POLL_TIME = 1200  # the first build is extremely slow if using docker-compose-*local*.yaml for bootstrap of builder image
+REGISTRY_URL = os.environ.get("MR_URL", "http://localhost:8080")
```

Comment:
on a totally unrelated note, I would use _analogous_ "injection approach" also for the other capability about https://github.com/kubeflow/model-registry/pull/421

Reply:
I would prefer to use the serialization capabilities already available on Pydantic with the addition of Python-dotenv, would you prefer to have this change applied to this PR or later?

Reply:
definitely for later.

I'm not opinionated on a library for bringing in the config values a-la Vyper or SmallRye Config, but we will need to support env variables, .env, and ConfigMaps, etc a-la 12Factor apps imho
@Al-Pragliola thoughts?

Reply:
Yes, we definitely need to support them as this approach makes everything more flexible 👍🏼